mods/sp/weapons/cabweapons.yaml

```diff
@@ -250,6 +255,12 @@ Dogzooka:
 			AircraftArmor: 100
 			ConcreteArmor: 5
 			Shield: 0
+		ValidTargets: Ground, Water
+		DamageTypes: Prone70Percent, TriggerProne, ExplosionDeath
+	Warhead@2Eff: CreateEffect
+		Explosions: sparks
+		ImpactSounds: rketinf1.aud
+		ExplosionPalette: apolra2
 
```

Comment:
Pretty sure this would look better with another palette, wasnt meant to be used for a explosion so probably try a gradient palette.

Reply:
ok, will do it later today.

Reply:
current gradient looks not very good.